python/src/aac/plugins/gen_plugin/templates/manifest_in.jinja2

```diff
@@ -0,0 +1,5 @@
+graft src
+graft tests
+
+include tox.ini
+include src/{{project.name}}/{{get_python_name(project.name)}}.aac
```

Comment:
The first project name reference cannot be left the way that it is. Believe there is a helper method for templates for putting in hyphens? A project name won't be left with spaces in the name for pathing.